docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -48,6 +49,326 @@ to connect to and issue SQL utilities, see the
 {docs-url}/client_installation/index.html[_{project-name} Client Installation Guide_].
 
 <<<
+[[cleanup_statement]]
+== CLEANUP Statement
+
+The CLEANUP Statement cleans up objects whose information is inconsistent in the metadata. This command can be used to clean up individual objects or all objects in a schema. This command can also be used to locate all inconsistent objects in the Trafodion metadata and then clean them up.
+
+Inconsistent objects can be cleaned up and removed from the metadata and HBase by using one of the following options.
+
+```
+CLEANUP {obj-type} {obj-name} [ ,UID {object-uid} ]

```

Comment:
Thanks Dave.
Yes, the {project-name} appears many times as a variable for Trafodion in .adoc files.
But here I'm afraid, the usage of curly brace, is also appropriate, as the Notation Conventions defined in the _Trafodion SQL Manaul_:
> { } Braces
Braces enclose **required syntax items**.
...
A group of items enclosed in braces is a list from which you are required to choose one item.

Here is an example of the syntax of GRANT Statement in the .adoc file and on the website:
``
GRANT {privilege [,privilege]... |ALL [PRIVILEGES]}
``
``
ON [object-type] [schema.]object
``
`` 
TO {grantee}
``
``
[WITH GRANT OPTION]
``
``
[[GRANTED] BY grantor]
``

As for this statement, `obj-type` and `obj-name` are required syntax items. :)